lib/gui/app/components/source-selector/source-selector.tsx

```diff
@@ -140,6 +139,9 @@ const URLSelector = ({
 			cancel={cancel}
 			primaryButtonProps={{
 				disabled: loading || !imageURL,
+				style: {
+					display: imageURL ? 'block' : 'none',
+				},
```

Comment:
Don't add the button to the document instead of hiding it with `display: none`

Reply:
This can't be done with current rendition

Reply:
Can we just keep the button but disable it ?

Reply:
Yes, but we haven't got any primary button disabled outside of the three on the main screen. Isn't it better to not show it unless there is a selection happening? It's pointless otherwise imho

Reply:
I don't like the cancel button moving